### PR TITLE
Update ChangeLog

### PR DESCRIPTION
**Pull request by Shreeshrii**
Add  major updates to 4.0.0alpha by Ray since Nov, 2016.

@zdenop, @stweil, @egorpugin @amitdo Please update with other major changes. Thanks!

### CODE REVIEW

**Comment by stweil** [comment]
\+ Remove support for VS2010.

**Comment by amitdo** [comment]
>+ Remove support for VS2010.

@egorpugin
Does Tesseract 4.0 built with CMake supports VS2010 or not?

**Comment by egorpugin** [comment]
No. Not out of the box.
One needs fixing some bugs in tiff library that comes from cppan.
Or there's always manual way of including dependencies. In this case VS2010 might work.

With cppan VS2015 and VS2017 are only supported.

**Comment by Shreeshrii** [comment]
+ Add Support for VS2015 and VS2017 with cppan
+ Implement invisible text only for PDF
+ Add AVX / SSE support for Windows
+ Require leptonica 1.74 or higher

**Comment by zdenop** [comment]
VS2010 has [limited support for C++11](https://msdn.microsoft.com/en-us/library/hh567368.aspx), so there should be more problems . I would expect that situation with VS2010 Express (free version of VS2010) is even worse.

**Comment by Shreeshrii** [comment]
Thanks, I have combined the updates for date range from Nov 2016 to March 2017. Suggest that any major commits also add to the changelog on regular basis.

Does semantic versioning requiring more frequent tagging?

@zdenop Please approve merge. Thanks!

**Comment by Shreeshrii** [comment]
@amitdo  I thought that you had made a similar list adding to release notes... Can't seem to find it now.

**Comment by amitdo** [comment]
Shree, here it is:
https://github.com/tesseract-ocr/tesseract/wiki/ReleaseNotes#in-development

**Comment by Shreeshrii** [comment]
Thanks.

- excuse the brevity, sent from mobile

File: ChangeLog [hunk 1]
```diff
@@ -1,7 +1,23 @@
-2016-11-11 - V4.00.00
+2016-11-11 to 2017-March - V4.00.00-alpha
```
Comment (zdenop) [inline ChangeLog]: Please do not change date format: use 2017-03-24 (instead of "2016-11-11 to 2017-March")

File: ChangeLog [hunk 2]
```diff
@@ -1,8 +1,5 @@
-2016-11-11 to 2017-March - V4.00.00-alpha
-  * Added new neural network system based on LSTMs, with major accuracy gains.
+2017-03-24 - V4.00.00-alpha-cf245f8
```
Comment (zdenop) [inline ChangeLog]: No! There is only one V4.00.00-alpha.  Please follow pattern of Changelog

**Pull request merged**